swift/ql/test/extractor-tests/generated/stmt/ForEachStmt/ForEachStmt_getNextCall.expected

```diff
@@ -0,0 +1,2 @@
+| for.swift:4:5:6:5 | for ... in ... where ... { ... } | for.swift:4:5:4:5 | call to next() |
+| for.swift:7:5:9:5 | for ... in ... { ... } | for.swift:7:5:7:5 | call to next() |
```

Comment:
The variadic pack case on line 13 seems to be missing from here and several of the other `.expected` files.  Is that expected, or a clue to the issue we were seeing with control flow in these packs perhaps???

Reply:
very good point, that's probably it!

Reply:
Great.  Happy with this remaining as follow-up work. :+1: